Bottom: aa49ac5bbe853bbc7d02c6bd6f11f4ca37041752
Top:    c1e3b0e7284978875258cdf2470d8ce9d9de1828
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-24 09:39:45 +0800

Refresh of autofs-5.0.7-fix-dead-LDAP-symbolic-link-when-LDAP-support-is-disabled.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 69fd1e7..fec5308 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -58,6 +58,7 @@
 - add after sssd dependency to unit file.
 - dont start readmap unless ready.
 - fix compilation of lookup_ldap.c without sasl.
+- fix dead LDAP symbolic link when LDAP support is disabled.
 
 25/07/2012 autofs-5.0.7
 =======================
diff --git a/modules/Makefile b/modules/Makefile
index 2589ae0..6e791cc 100644
--- a/modules/Makefile
+++ b/modules/Makefile
@@ -76,7 +76,9 @@ install: all
 	-rm -f $(INSTALLROOT)$(autofslibdir)/mount_smbfs.so
 	ln -fs lookup_file.so $(INSTALLROOT)$(autofslibdir)/lookup_files.so
 	ln -fs lookup_yp.so $(INSTALLROOT)$(autofslibdir)/lookup_nis.so
+ifeq ($(SASL), 1)
 	ln -fs lookup_ldap.so $(INSTALLROOT)$(autofslibdir)/lookup_ldaps.so
+endif
 	ln -fs mount_nfs.so $(INSTALLROOT)$(autofslibdir)/mount_nfs4.so
 ifeq ($(EXT2FS), 1)
  ifeq ($(EXT3FS), 1)
